networkx/utils/backends.py

```diff
@@ -1862,21 +1873,29 @@ def _convert_graph(
                 return graph
             backend = _load_backend(graph.__networkx_backend__)
             rv = backend.convert_to_nx(graph)
+            func = backend.convert_to_nx
```

Comment:
I think this makes `backend.convert_to_nx` run twice. Did you mean to replace line 1875?

Also, when would this code run?   I mean, when is `backend_name == "networkx"`?
I thought that meant we are using networkx as the backend, so converting would not be needed. I'm perhaps forgetting something...

Reply:
Ha, good catch! I forgot to delete the previous line `rv = backend.convert_to_nx(graph)`, so this should only be called once.

When the backend is `"networkx"`, we ensure all graph inputs are networkx graphs by calling `backend.convert_to_nx`.